arangod/GeneralServer/SslServerFeature.h

```diff
@@ -59,17 +59,33 @@ class SslServerFeature : public application_features::ApplicationFeature {
   void unprepare() override final;
   virtual void verifySslOptions();
 
-  virtual std::shared_ptr<asio_ns::ssl::context> createSslContext();
+  std::unique_ptr<asio_ns::ssl::context> createSslContextInternal(std::string keyfileName,
```

Comment:
If this can be private, it should be :) and by its name it should also be `const`...

Reply:
Right. Done.